CustomApps/lyrics-plus/OptionsMenu.js

```diff
@@ -218,8 +218,9 @@ const AdjustmentsMenu = react.memo(({ mode }) => {
 					],
 					onChange: (name, value) => {
 						CONFIG.visual[name] = value;
+						console.log(CONFIG.visual, APP_NAME, name, value);
 						localStorage.setItem(`${APP_NAME}:visual:${name}`, value);
-						name === "delay" && localStorage.setItem(`lyrics-delay:${Spicetify.Player.data.track.uri}`, value);
+						name === "track-delay" && localStorage.setItem(`lyrics-track-delay:${Spicetify.Player.data.track.uri}`, value);
```

Comment:
Keep `track-delay` as `delay`. There are already saved configurations of each individual track delays set by users.
Doing this would mean leaving them unused and essentially resetting all saved values.